docs/swagger.yaml

```diff
@@ -0,0 +1,978 @@
+swagger: "2.0"
+info:
+  description: "This is a sample AutoML server.  You can find out more about     Swagger at [http://swagger.io](http://swagger.io) or on [irc.freenode.net, #swagger](http://swagger.io/irc/).      For this sample, you can use the api key `special-key` to test the authorization     filters."
+  version: "1.0.0"
+  title: "AutoML"
+  termsOfService: "http://swagger.io/terms/"
+  contact:
+    email: "apiteam@swagger.io"
+  license:
+    name: "Apache 2.0"
+    url: "http://www.apache.org/licenses/LICENSE-2.0.html"
+host: "automl.swagger.io"
+basePath: "/v2"
+tags:
+- name: "Authentication"
+  description: "API Schema of AutoML"
+  externalDocs:
+    description: "Find out more about AutoML"
+    url: "http://swagger.io"
+- name: "Dataset Management"
+  description: "Management of Dataset"
+- name: "Train"
+  description: "Operations about Training"
+- name: "Evaluate"
+  description: "Operations about Evaluation"
+  externalDocs:
+    description: "Find out more about AutoML"
+    url: "http://swagger.io"
+- name: "Predict"
+  description: "Operations about Prediction"
+schemes:
+- "https"
+- "http"
+paths:
+  /automl/login:
+    post:
+      tags:
+      - "Authentication"
+      summary: "Login to AutoML"
+      description: ""
+      operationId: "login"
```

Comment:
I think we need register as well.

Reply:
Absolutely agree! And we need to think carefully about the multi-tenancy case.